src/TreeMenu.js

```diff
@@ -78,6 +78,7 @@ const Menu = ({
      */
     resRenderGroup.push(
         resources.filter(r => r.options && !r.options.hasOwnProperty('menuParent') && !r.options.hasOwnProperty('isMenuParent'))
+            .filter(r => r.hasList)
```

Comment:
This is good! But filtering the resources with hasList after doing the parent-related checks won't do us much good. Instead it would be a good idea to filter by hasList first and then go on with the parent checks as that way we'd be making sure we don't fo parent checks on resources which don't have a list at all.